load-balancer/api-docs/common-gs/auth-using-curl.rst

```diff
@@ -137,7 +137,7 @@ request, prefix the variable name with a ``$``, for example ``$ENDPOINT``.
 
    .. code::
 
-      $ export TENANT_ID="tenant-id"
+      $ export ACCOUNT_ID="account-id"
```

Comment:
Line 67 says  tenant_id is the account number, i think we  can keep it as tenant_id here.